.landscaper/gardenlet/example/landscaper-resources/component_descriptor.yaml

```diff
@@ -0,0 +1,34 @@
+# Component descriptor required by the Landscaper.
```

Comment:
Hm, similar question like above in the component descriptor file.
Why does this file look different, actually?

Reply:
This is the Component descriptor required by the Landscaper.
I think in the future this should be generated by the landscaper out of the box, but right now is still required to be created manually.
Right @schrodit ?